tests/endpoints/test_voyages_search_enriched.py

```diff
@@ -98,10 +98,13 @@ def test_search_from_description_raw(self):
         )
 
         assert len(res) > 0
-        assert res[0].schema_version == "1.0.0"
         assert res[0].vessel is not None
         assert len(res[0].events) > 0
         assert res[0].events[0].event_group is not None
+        assert res[0].start_timestamp is not None
```

Comment:
Adding assert for the properties that should never be nullable, and removing the hardcoded schema version.